build.gradle

```diff
@@ -25,7 +25,7 @@ plugins {
   id 'com.github.ben-manes.versions' version '0.52.0'
   id 'com.github.jk1.dependency-license-report' version '2.9'
   id 'io.spring.dependency-management' version '1.1.7'
-  id 'net.ltgt.errorprone' version '4.1.0' apply false
```

Comment:
do we still want to use the apply false?

Reply:
i have no idea what it even does

Reply:
We should keep using it. It is just a workaround to prevent potential classpath issues and loading this multiple times. I don't remember the details, but because error-prone ties into the compilation of the project, bad things can happen if you end up loading it multiple times.

TLDR This is a simple way to prevent it, and we probably don't want to remove it unless we really know what we are doing :D